view_acl/utils.py

```diff
@@ -0,0 +1,27 @@
+import inspect
+
+def get_caller_name(skip=2):
```

Comment:
Does it get package hierarchy for the view? We have to recognize packages in order to distinguish views for different applications